src/App.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
-import HomePage from './Pages/HomePage';
-import ShopPage from './Pages/ShopPage';
-import { SignInPage } from './Pages/SignInPage/SignInPage';
-import { CartPage } from './Pages/CartPage';
-import { WishListPage } from './Pages/WishListPage';
-import { RequireAuth } from './Components/RequireAuth/RequireAuth';
+import { HomePage } from './export';
```

Comment:
This can be put into a file (which is already done) and then imported from that one file just once instead of importing from the same file repeatedly.

Reply:
Thank you Done